tldw_Server_API/app/api/v1/endpoints/watchlists.py

```
@@ -4998,6 +5063,87 @@ async def list_templates(
     return WatchlistTemplateListResponse(items=items)
 
 
+@router.post(
+    "/templates/validate",
+    response_model=TemplateValidationResult,
+    summary="Validate Jinja2 template syntax",
+)
+async def validate_template(
+    payload: TemplateValidateRequest,
+    current_user: User = Depends(get_request_user),
```

Ruff (0.15.1)

[warning] Unused function argument: `current_user`

(ARG001)